bluepyopt/ephys/locations.py

```diff
@@ -392,6 +393,35 @@ def instantiate(self, sim=None, icell=None):
         return self.find_icomp(sim, branches)
 
 
+class NrnTrunkSomaDistanceCompLocation(NrnSecSomaDistanceCompLocation):
+    """Location at a distance from soma along a main direction.
+
+    This is most useful to follow the trunk of an apical dendrite
+    without knowing the apical point, but only that apical trunk goes along y.
+    """
+
+    def set_sec_index(self, icell=None, direction=[0.0, 1.0, 0.0]):
```

Comment:
Lists should normally not be used as default values (see e.g. https://docs.python-guide.org/writing/gotchas/). You could set it to None, and immediately assign it to this value on the first line of the method under a 'if direction is None' statement

Reply:
ah yes, I keep forgetting that, but in some codes, lint complains about it, then I don't forget